pkg/config/args_authorization.go

```diff
@@ -0,0 +1,12 @@
+package config
+
+import (
+	"github.com/spf13/pflag"
+)
+
+var ProxyLocalAuthorization bool
+
+func AddProxyAuthorizationFlags(set *pflag.FlagSet) {
+	set.BoolVarP(&ProxyLocalAuthorization, "proxy-local-authorization", "", false,
+		"do authorization locally before proxy request")
```

Comment:
`perform an additional delegated authorization against the hub cluster for the target proxying path when invoking "clustergateway/proxy" subresource`

Reply:
It is fine. Let me change it. Thank you.